joomla/joomla-cms
# com_templates removal of PHP warnings

Pull Request for Issue #31747

### Summary of Changes
Copy of #27851 for staging , credit goes to @Quy 

### Testing Instructions
as per Issue #31747, apply patch 
Search in page source for id="resizeModal" , should not be there

### Actual result BEFORE applying this Pull Request
PHP Warning: Trying to access array offset on value of type null
logs have PHP Warning: Trying to access array offset on value of type null

### Expected result AFTER applying this Pull Request
no more PHP Warnings in \administrator\components\com_templates\tmpl\template\default_modal_resize_body.php

### Documentation Changes Required
n/a
